src/lib.rs

```diff
@@ -143,6 +145,15 @@ pub trait LabelBuilder<'a> {
     fn y_label_format(&'a mut self, format: LabelFormat) -> &'a mut Chart<'a>;
 }
 
+/// Provides an interface for adding tick labels to the y-axis
+pub trait TickLabelBuilder<'a> {
+    // Horizontal labels don't allow for support of x-axis tick labels
+    /// Specifies the tick label density of y-axis.
+    /// YAxisTickLabels::Sparse will change the canvas height to the nearest multiple of 16
+    /// YAxisTickLabels::Dense will change the canvas height to the nearest multiple of 8
+    fn y_tick_labels(&'a mut self, density: YAxisTickLabels) -> &'a mut Chart<'a>;
```

Comment:
Not sure about the name.. maybe `y_tick_display`?

Reply:
I changed it to `y_tick_display`. I originally chose the name to try to avoid confusion with the private function `format_y_axis_tick`.